internal/database/atendee_table.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+    "database/sql"
+    "fmt"
+    "log"
+
+    _ "github.com/go-sql-driver/mysql"
+)
+
+func CreateAttendeesTable() {
+    // Connect to the database
+    dsn := "root:admin123@tcp(localhost:3306)/events" //dsn
+    db, err := sql.Open("mysql", dsn) //open connection to db
+    if err != nil { //check for errors
+        log.Fatal(err) //exit if error is found
+    }
+    defer db.Close() //close the connection when the function ends
+	
+    // SQL query to create the attendees table
+    createTableQuery := `
+    CREATE TABLE IF NOT EXISTS attendees (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        event_id INT,
```

Comment:
I think we can remove the event_id on this table; the rsvp table will eventually link the attendee to the event (since one attendee could RSVP for multiple different events)!